docs/README.md

```diff
@@ -70,8 +70,8 @@ If you want to learn more, more video tutorials are [there,](https://www.youtube
 
 **Getting Started**
 
-Without version specified, the version of the server will be guessed automatically, you can set a specific one using the version option.
-For example `version:"1.8"`.
```

Comment:
why remove the version example ?

Reply:
its a few lines down still there, just not really as important since mineflayer can see what version it needs, which is all that most users need